improved OFF brand validation

5h3rr1ll opened the pull request:
improved OFF brand validation by adding logic to the create create_feedback_string()-function and changed the is_big_ten_by_name()

5h3rr1ll committed:
refractored is_big_ten function

5h3rr1ll committed:
adaped test to new function name

5h3rr1ll committed:
adapted urls to new function name

Darjusch commented on goodbuyDatabase/urls.py:
```diff
@@ -32,9 +32,9 @@
-        name="is_in_own_database",
+        "is_product_db/<str:code>/",
+        goodbuyDatabase_endpoints.is_product_db,
+        name="is_product_db",
```
endpoint sounds a bit weird.
maybe is_product_in_db?

5h3rr1ll replied:
I tried to shorten the name bit but okay, we can put an "in" in there. ;)

Darjusch commented on goodbuyDatabase/endpoints.py:
```diff
@@ -80,16 +74,31 @@ def create_feedback_string(request, product_object):
-    is_big_ten_answer = is_big_ten(request, code=product_object.code)
-    product_serialized["is_big_ten"] = is_big_ten_answer
-    return product_serialized
+    if product_object.data_source == "1":
```
Readability würde verbessert werden wenn da OFF oder OpenFoodFacts stehen würde statt 1. 
Vielleicht als kleiner fix einfach ne variable dadrüber erstellen mit dem value 1?

5h3rr1ll replied:
Ja, verstehe ich, aber eine Variable zu erzeugen, die dann nicht mal wie ne variable benutzt wird, da sie Ihren wert nicht ändert, finde ich irgendwie strange. Aber okay, ändere ich.

Darjusch commented on goodbuyDatabase/endpoints.py:
```diff
@@ -24,36 +24,30 @@
+        return "We don't know"
+    try:
+        brand = Brand.objects.get(name=brand_name)
+        return BigTen.objects.filter(name=brand.name).exists()
```
I am a bit confused shouldn't this also be brand.corporation? 
In BigTen are only Corporation names.

5h3rr1ll committed:
Update urls.py

Changed is_product_db to is_product_in_db

5h3rr1ll committed:
Update endpoints.py

changed function name from is_product_db to is_product_in_db

5h3rr1ll committed:
Update endpoints.py

Line 36 to 39: didn't only change brand.name to brand.corporation, also added an if condition, since it needs to be checked if the brand has a corporation at all.

5h3rr1ll committed:
Update endpoints.py

Line 87: changed 'if product_object.data_source == "1"' to 'if product_object.data_source == open_food_facts' for readabilty.

5h3rr1ll committed:
Update urls.py

Line 36: changed is_product_db to is_product_in_db
